proposals/lip-0045.md

```diff
@@ -476,7 +503,8 @@ ownChainAccountSchema = {
             "fieldNumber": 1
         },
         "ID": {
```

Comment:
I think `chainID` would be more clear. Note that in the description below, you already assume that this is the property name.